ui/admin/app/components/form/auth-method/ldap/index.hbs

```diff
@@ -25,7 +25,6 @@
   <Hds::Form::TextInput::Field
     @isOptional={{true}}
     @value={{@model.name}}
-    @isInvalid={{@model.errors.name}}
```

Comment:
Good question - `isInvalid` is used at the field level to indicate incorrect fields. In this case, we removed it because the API doesn't send us field-level errors for most fields in this form.

To answer your question about how to mark the form as an error- this is the usual process of showing the error message on save at the bottom of the page.

The only thing you won't see by this change is the field highlighted in red with a field-specific message under the field.